### PR TITLE
update toml

### PR DESCRIPTION
**Pull request by emmakbat**

### CODE REVIEW

File: src/qnngds/cells.py [hunk 1]
```diff
@@ -334,9 +336,14 @@ def nanowires(
     NANOWIRES = Device(f"NWIRES({cell_text})")
     nanowires_ref = []
     for i, channel_source_w in enumerate(channels_sources_w):
-        nanowire_ref = NANOWIRES << device.nanowire.spot(
-            channel_source_w[0], channel_source_w[1]
-        )
+        if lengths is None:
```
Comment (reed-foster) [inline src/qnngds/cells.py:339]: probably want to uncomment the else?

Comment (emmakbat) [reply src/qnngds/cells.py:339]: Yes sorry -- before I had things set up correctly my code couldn't "see" the variable_lengths function so I was unable to test this part of the code, and I commented it out to be safe. Currently working on testing the else and adding a couple other improvements

**Comment by reed-foster** [comment]
I'm not sure why none of the github actions ran on this. Looks like it's cause this is from a fork, so you're considered a first-time contributor: https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks

I'll see if I can have it run the actions.

File: src/qnngds/cells.py (hunk 1 above)
Comment (reed-foster) [reply src/qnngds/cells.py:339]: sounds good, let me know when the PR is ready

**Comment by emmakbat** [comment]
Yeah, forking the repository was the recommended workflow before the refactoring. I'm not totally sure how to reset things on my machine so it's not working from the fork

**Comment by reed-foster** [comment]
Seems like I just need to add `pull_request` event for the `on` section of the github action: https://stackoverflow.com/questions/73314842/unable-to-trigger-github-actions-when-pull-request-comes-from-a-forked-repositor

**Comment by emmakbat** [comment]
Updated, and I also added a "tolerance" option to DieParameters that shrinks the pads to allow for MLA alignment error (which is inevitable, so accounting for it should be the default option)

File: docs/user/images/devices/resistor/meander_sc_contacts.png
Comment (reed-foster) [inline docs/user/images/devices/resistor/meander_sc_contacts.png:1]: not sure why this changed lol

File: src/qnngds/cells.py [hunk 2]
```diff
@@ -302,6 +302,8 @@ def nanowires(
     outline_dev: Union[int, float] = 0.5,
     device_layer: int = 1,
     text: Union[None, str] = None,
+    lengths: List[float] = None,
+    tolerance: float = 5,
```
Comment (reed-foster) [inline src/qnngds/cells.py:306]: should `tolerance` be in `die_parameters`?

Comment (reed-foster) [reply src/qnngds/cells.py:306]: I see you already have `pad_tolerance` in `DieParameters`

**Review by reed-foster — APPROVED** [review]

**Pull request closed without merging**